src/components/ui-base/constant.ts

```diff
@@ -12,4 +12,13 @@
  */
 
 // parent app my turn on or off the chakra-ui components with this flag
-export const isChakraRemoved = process.env.REACT_APP_ENABLE_CSS || false;
+
+function getEnvVariable(varName: string) {
```

Comment:
a more optimized approach would be to add a second param called "default", and then move this function into a utility file so it can be reused in the future, optionally refactor api.ts to use it.

Reply:
good suggestions, will refactor into a util with default.